apps/docs/content/components/Avatar.mdx

```diff
@@ -0,0 +1,124 @@
+---
+title: Avatar
+status: Experimental
+figma: 'https://www.figma.com/file/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=543%3A19830&t=Be91VGjX2BANJnQc-11'
+source: https://github.com/primer/brand/tree/main/packages/react/src/Avatar/Avatar.tsx
+storybook: '/brand/storybook/?path=/story/components-Avatar--playground'
```

Comment:
nit-pick 🤏 : Shouldn't this link go to the default story instead?

Reply:
I'm unsure what the rule is; some components point to the playground story, while others point to the default one. Any thoughts, @rezrah?

Reply:
Yes, should be default. If it's pointing at playground, it's an oversight.